Galiojanti suvestinė redakcija (nuo 2019-07-01)

kasperaitis opened the pull request:
Dėl Teisės užsiimti radijo mėgėjų veikla suteikimo tvarkos ir užsiėmimo šia veikla sąlygų aprašo patvirtinimo.

12 punktas:
Leidimas užsiimti radijo mėgėjų veikla išduodamas 10 metų terminui, išskyrus Aprašo 19 punkte nurodytą atvejį.

kasperaitis committed:
Galiojanti suvestinė redakcija (nuo 2019-07-01)

Dėl Teisės užsiimti radijo mėgėjų veikla suteikimo tvarkos ir užsiėmimo šia veikla sąlygų aprašo patvirtinimo.

12 punktas:
Leidimas užsiimti radijo mėgėjų veikla išduodamas 10 metų terminui, išskyrus Aprašo 19 punkte nurodytą atvejį.

kasperaitis committed:
Atnaujinti galimi atsakymai.

Aktuali nuo 2020 m. vasario 11 d.

va1da5 commented:
Galbūt galėčiau paprašyti pridėti šiuos pakeitimus į `questions.yml` faila. Būtent iš jo vėliau yra automatiškai generuojamas JSON failas, kuriam šis PR'as daro pakeitimus.

kasperaitis committed:
Atsakymo pataisymas.

Dėl Teisės užsiimti radijo mėgėjų veikla suteikimo tvarkos ir užsiėmimo šia veikla sąlygų aprašo patvirtinimo.

12 punktas:
Leidimas užsiimti radijo mėgėjų veikla išduodamas 10 metų terminui, išskyrus Aprašo 19 punkte nurodytą atvejį.

va1da5 reviewed (approved):
Ačiū.
